src/raylib.h

```diff
@@ -415,8 +415,9 @@ typedef enum {
     FLAG_WINDOW_RESIZABLE   = 4,    // Set to allow resizable window
     FLAG_WINDOW_UNDECORATED = 8,    // Set to disable window decoration (frame and buttons)
     FLAG_WINDOW_TRANSPARENT = 16,   // Set to allow transparent window
-    FLAG_MSAA_4X_HINT       = 32,   // Set to try enabling MSAA 4X
-    FLAG_VSYNC_HINT         = 64    // Set to try enabling V-Sync on GPU
+    FLAG_WINDOW_HIDDEN      = 32,   // Set to create the window initially hidden
```

Comment:
Some people prefer to use raylib as shared library and to avoid complicating
their raylib experience, one should be wary of subtle ABI breaks.
With this change, an old program linking against a new shared raylib would have the window hidden instead of MSAA 4x!
It would've been better to use `FLAG_WINDOW_HIDDEN = 128` here, you could reorder it to have the FLAG_WINDOW*s grouped together

Reply:
@a3f I agree, despite being a small aesthetic change I could be very annoying, I'll revert it.

Reply:
Reverted in commit https://github.com/raysan5/raylib/commit/25205b5113587af71af0f39549c9bb0a3c1faf49